lib/aruba/environments/unix_environment.rb

```diff
@@ -0,0 +1,11 @@
+require 'aruba/environments/basic_environment'
+
+module Aruba
+  module Environments
+    class UnixEnvironment < BasicEnvironment
+      def match?(*)
+        Aruba::Platform.unix?
```

Comment:
Making that change doesn't work.  With that change in place, when running on Windows, both ```UnixEnvironment.match``` and ```WindowsEnviornment.match``` return ```true``` which leads to the wrong ```Environment`` being created with ``` super environments.find(&:match?)```.
Only *one* of the classes should return true at any given time.  (I tried it on my system to be sure before committing.)  
Unless I'm missing something about this.